.changesets/config_helm_extra_labels.md

```diff
@@ -0,0 +1,5 @@
+### Helm extra labels
+
+Allow chart users to add custom labels to the router deployment and its pods.
+
+By [@gscheibel(https://github.com/gscheibel/) in https://github.com/apollographql/router/pull/2903
```

Comment:
```suggestion
### Helm: Charts now support for `extraLabels` for Deployments/Pods

Our Helm charts now support a new value called `extraLabels`, which enables chart users to add custom labels to the Router Deployment and its Pods.

By [@gscheibel(https://github.com/gscheibel/) in https://github.com/apollographql/router/pull/2903
```

Reply:
I committed a slightly modified version of this